tests/python/codegen/test_target_codegen_aarch64.py

```diff
@@ -700,5 +700,31 @@ def before(a: T.handle):
     assert "get.active.lane.mask" in ll
 
 
+@pytest.mark.skipif(
```

Comment:
This validates that we have the correct output for architectures that support SVE, but it doesn't test the behavior of other targets that do not (yet) support predicated loads/stores.  While the `VectorizeLoop` pass would only insert a predicated load/store for targets that support it, the predicated load/store could still be generated in hand-written kernels, or through other transforms in the future.

Can we add a test, parametrized over each target tested in CI, which attempts to compile a PrimFunc containing predicated loads/stores?  For each target that supports sve, `tvm.build` should compile without error, and for each target that does not, `tvm.build` should raise an exception.

Reply:
Done